src/topology/metric_space/basic.lean

```diff
@@ -456,6 +462,17 @@ show dist x x ≤ ε, by rw dist_self; assumption
 @[simp] lemma closed_ball_eq_empty : closed_ball x ε = ∅ ↔ ε < 0 :=
 by rw [← not_nonempty_iff_eq_empty, nonempty_closed_ball, not_le]
 
+/-- A ball of radius zero is the sphere of radius zero -/
+theorem closed_ball_zero_eq_sphere_zero : closed_ball x 0 = sphere x 0 :=
+set.ext $ λ _, iff.symm $ le_antisymm_iff.trans $ and_iff_left dist_nonneg
```

Comment:
Is there much point in having this one given the next?

Reply:
No, I guess not